### PR TITLE
feat: update to egg 0.7

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: boilerplate/_package.json [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
   "dependencies": {
-    "egg": "^0.6.3"
+    "egg": "^0.7.0"
   },
   "devDependencies": {
     "autod": "^2.7.1",
     "egg-bin": "^1.7.0",
     "egg-ci": "^1.1.0",
-    "egg-mock": "^1.2.0",
+    "egg-mock": "2",
```
Comment (popomore) [inline boilerplate/_package.json:13]: 用 autod 生成吧

Comment (atian25) [reply boilerplate/_package.json:13]: @dead-horse autod 支持下配置 package.json 的文件名? boilerplate 里面用来更新 _package.json 文件方便点

**Review by atian25 — COMMENTED** [review]
+1

**Comment by atian25** [comment]
1.1.0

**Pull request merged**